107/src/main/java/org/ehcache/jsr107/Eh107CacheManager.java

```diff
@@ -64,10 +65,12 @@
   private final Eh107CacheWriterFactory cacheWriterFactory;
   private final Jsr107Service jsr107Service;
   private final org.ehcache.config.Configuration ehConfig;
+  private final Eh107CacheService eh107CacheService;
```

Comment:
Why introduce a new service here? Why not have `Jsr107Service implements EhcacheService`?